### PR TITLE
Psf offsets

### LINKED ISSUE #37: optionally return offset for psf image
This is needed for the psf coaddition

### PR DESCRIPTION
**Pull request by esheldon**
closes #37

### CODE REVIEW

**Comment by beckermr** [comment]
See my comment. This PR is not needed because the user already has the info they need for this. The offset is simply `x - int(x+0.5), y - int(y+0.5)`.

File: descwl_shear_sims/simple_sim.py [hunk 1]
```diff
@@ -706,10 +706,15 @@ def _psf_render_func(*, x, y, center_psf):
                     x=x-int(x+0.5),
                     y=y-int(y+0.5),
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py]: The offset is here and directly computed from the inputs. No need to send it back out.

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: I want this available so I don't have to hard code `x=x-int(x+0.5)` in other codes that use the offset.  What if conventions change and the offset is calculated differently in the future? etc.

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: Also, it does no harm to have it there.

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py]: This is the convention for a pixel center. It won't change and hasn't in probably 20+ years. :)

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py]: Feel free to merge if you want though.

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: A real case where the convention could be different:  what if the stack uses pixel centers that are 0.5 different from galsim?

**Pull request merged**